utils/sender/src/main.rs

```diff
@@ -36,6 +38,27 @@ const SMALL_TOKEN_AMOUNT: Value =
 const BIG_TOKEN_AMOUNT: Value =
 	Value { value: ValueDef::Primitive(Primitive::U128(u128::MAX)), context: () };
 
+fn spawn_healthcheck_server(port: u16) {
+	let addr = SocketAddr::from(([0, 0, 0, 0], port));
+
+	std::thread::spawn(move || {
+		let runtime = tokio::runtime::Builder::new_current_thread()
+			.enable_all()
+			.build()
+			.unwrap();
+
+		runtime.block_on(async move {
+			let app = Router::new().route("/health", get(|| async { StatusCode::OK }));
```

Comment:
@sandreim do you want me to ping the chain here and check that we are actually connected? It is not possible to start the sender without being connected to a node.... but we can lose connection over the course of the `sender` binary running. Do you have any proposal for easiest way to ping node connection here?

Reply:
Changed to do a request against node